packages/contracts/src/systems/AutoBattleSystem.sol

```diff
@@ -143,6 +209,12 @@ contract AutoBattleSystem is System {
         // clear player if it's defeated, update finishedBoard if else
         if (playerHealth == 0) {
             Utils.clearPlayer(_gameId, _player);
+
+            // //  clear bot
+            bool isSinglePlay = Game.getSingle(_gameId);
+            if (isSinglePlay) {
+                Utils.clearPlayer(_gameId, Utils.generateRandomAddress(_player));
+            }
```

Comment:
It think these code is not necessary. If there is not one player, the game will end.

Reply:
In fact the enemy will not fail. Next time when entering again, the enemy board is not cleared

Reply:
> In fact the enemy will not fail. Next time when entering again, the enemy board is not cleared

See https://github.com/HelheimLabs/autochessia/pull/162#discussion_r1318037200